src/components/NotesComponent/NotesComponent.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import ListComponent from '../ListComponent/ListComponent';
+import SearchComponent from '../SearchComponent/SearchComponent';
+
+const notes = [
```

Comment:
This is good :)

Reply:
This is mocked content, so it's okay for it to be in lowercase, however I believe it shouldn't be defined in this task at all - so it will be removed in the end